config/anycable.yml

```diff
@@ -9,8 +9,7 @@
 #
 # Read more about AnyCable configuration here: https://docs.anycable.io/v1/#/ruby/configuration
 #
-default: &default
-  # Turn on/off access logs ("Started..." and "Finished...")
+default: &default # Turn on/off access logs ("Started..." and "Finished...")
```

Comment:
Revert

Reply:
I can't, prettier changes this automatically